### PR TITLE
Feature/manager

### LINKED ISSUE #7: Adjust site for various screen sizes
### Media Query Manager:

- **0-600px**: Phone,
- **600-900px**: Tablet portrait,
- **900-1200px**: Tablet landscape,
- **1200 -1800px**: Where normal styles apply,
- **1800px+**: Bigger desktops

### PR DESCRIPTION
**Pull request by demaceo**
### What does  this PR do?
After logging in with the manager's credentials, a user can now view:
- each room's information.
- each customer.

### Any background context you want to provide?
- Planning on recycling the HTML elements for the **Manager View** tabs so that I won't have to reinvent the wheel when building out the **Customer History** & **Manage Bookings** sections.
- Need to add further functionality for the **Room Statuses** & **Customer History** sections so that they display all of the appropriate data necessary.

### What are the relevant tickets?
closes #7

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -4,10 +4,13 @@
 // }
 // $primary-background: radial-gradient(circle, rgba(0,0,0,0) 15%, rgba(0,0,0,0) 1%);
 // $palmys: url(./images/palmys.jpg);
+$ice-blue: #D7DFE0;
```
Comment (Estaffieri) [inline src/css/base.scss:7]: Oops! I definitely called my variables like "fireweed" and "cappuccino".

Comment (demaceo) [reply src/css/base.scss:7]: I love that 😂

File: src/index.js [hunk 2]
```diff
@@ -30,64 +31,69 @@ let passwordInput = document.querySelector('#password');
...
 
-// --- FUNCTIONS ---
+let customerHistoryNavLink = document.querySelector('#customer-history-nav');
+let customerHistorySection = document.querySelector('.customer-history-section');
+let customerHistoryBlock = document.querySelector('.customer-history-block');
+let searchUsersInput = document.querySelector('.search-input');
 
+let manageBookingsNavLink = document.querySelector('#manage-bookings-nav');
+let manageBookingsSection = document.querySelector('.manage-bookings-section');
+let manageBookingsForm = document.querySelector('.manage-bookings-form');
```
Comment (Estaffieri) [inline src/index.js:65]: You know that domLoader.js file that I've started building? It turns into a place to put all querySelectors and just import them each as variables into index.js to make your entry point less top heavy.

**Review by Estaffieri — APPROVED** [review]
Very nice job! The styling elements look like they are going fun places and I see you've added mostly vh and vw for your elements!  This will really help your site be more responsive and easier to use in the long run!

File: src/index.js (hunk 2 above)
Comment (demaceo) [reply src/index.js:65]: WELP! I was fully unaware of that! Definitely will look into that

**Pull request merged**